reporters/cloud-run-scripts/CloudRunnerHub.js

```diff
@@ -69,34 +68,69 @@ app.post("/", async (req, res) => {
     // Loop over all config objects in the config file and for each append a call promise to the promiseArray.
     let promiseArray = [];
     for (const botName in configObject) {
-      const botConfig = appendBlockNumberEnvVars(configObject[botName], lastQueriedBlockNumber, latestBlockNumber);
-      console.log("Appending to promise array", botName); // TODO: refine this logging with a winston log
-      // TODO: when running locally we can execute a JSON call to a local restful API. Refactor this for intergration tests.
-      // promiseArray.push(_postJson(process.env.PROTOCOL_RUNNER_URL, botConfig));
-      promiseArray.push(_executeCloudRun(process.env.PROTOCOL_RUNNER_URL, botConfig));
+      const botConfig = _appendBlockNumberEnvVars(configObject[botName], lastQueriedBlockNumber, latestBlockNumber);
+      if (process.env.ENVIRONMENT == "production") {
+        promiseArray.push(_executeCloudRun(process.env.PROTOCOL_RUNNER_URL, botConfig));
+      } else {
+        promiseArray.push(_postJson(process.env.PROTOCOL_RUNNER_URL, botConfig));
+      }
     }
+    Logger.debug({
+      at: "CloudRunnerHub",
+      message: "Executing cloud run query from config file",
+      lastQueriedBlockNumber,
+      latestBlockNumber,
+      botsExecuted: Object.keys(configObject)
+    });
 
     // Loop through promise array and submit all in parallel. `allSettled` does not fail early if a promise is rejected.
+    // This `results` object will contain all information sent back from the spokes. This contains the process exit code,
+    // and importantly the full execution output which can be used in debugging.
     const results = await Promise.allSettled(promiseArray);
-    console.log(JSON.stringify(results)); // TODO: refine this logging with a winston log
+
+    Logger.debug({
+      at: "CloudRunnerHub",
+      message: "Batch execution promise resolved",
+      results
+    });
 
     // Validate that the promises returned correctly. If ANY have error, then catch them and throw them all together.
```

Comment:
This file chane is the main part of this PR: dealing with errors thrown from the spokes in a more consistent and robust way.